packages/sysutils/busybox/config/busybox-init.conf

```diff
@@ -33,7 +33,7 @@ CONFIG_FEATURE_PREFER_APPLETS=y
 CONFIG_BUSYBOX_EXEC_PATH="/proc/self/exe"
 # CONFIG_SELINUX is not set
 # CONFIG_FEATURE_CLEAN_UP is not set
-CONFIG_FEATURE_SYSLOG=y
+# CONFIG_FEATURE_SYSLOG is not set
```

Comment:
intentional change?

Reply:
Yes, busybox `make oldconfig`  disabled it - syslog is only being enabled by [FEATURE_MOUNT_NFS](https://github.com/mirror/busybox/blob/1303962957fb900ed97c5958403990e885b06e29/util-linux/mount.c#L68-L79):
```
//config:config FEATURE_MOUNT_NFS
//config:	bool "Support mounting NFS file systems on Linux < 2.6.23"
//config:	default n
//config:	depends on MOUNT
//config:	select FEATURE_SYSLOG
//config:	help
//config:	Enable mounting of NFS file systems on Linux kernels prior
//config:	to version 2.6.23. Note that in this case mounting of NFS
//config:	over IPv6 will not be possible.
//config:
//config:	Note that this option links in RPC support from libc,
//config:	which is rather large (~10 kbytes on uclibc).
```